client/components/TestPlanReportStatusDialog/index.jsx

```diff
@@ -39,6 +40,28 @@ const TestPlanReportStatusDialog = ({
         [testPlanVersion]
     );
 
+    // section:
+    // Declare array of all reports
+    const allReports = ats
+        .map(at =>
+            at.browsers.map(browser => ({
+                browser: { name: browser.name, id: browser.id },
+                at: { name: at.name, id: at.id }
+            }))
+        )
+        .flat();
```

Comment:
I noticed this code seems to be failing in the tests, any idea about that?

Reply:
It might be the `ats` prop that I added to the component.

Reply:
Nice job fixing the tests 👍